Makefile

```
@@ -372,14 +374,20 @@ credentials/pull-secret.json:
 
 .PHONY: lifecycle-agent-deploy
 lifecycle-agent-deploy: lifecycle-agent
-	KUBECONFIG=../$(SNO_KUBECONFIG) \
-	IMG=$(LCA_IMAGE) \
-		make -C lifecycle-agent install deploy
-	@echo "Waiting for deployment lifecycle-agent-controller-manager to be available"; \
-	until $(oc) wait deployment -n openshift-lifecycle-agent lifecycle-agent-controller-manager --for=condition=available=true; do \
-		echo -n .;\
-		sleep 5; \
-	done; echo
+	export KUBECONFIG=../$(SNO_KUBECONFIG); \
```

```suggestion
	@export KUBECONFIG=../$(SNO_KUBECONFIG); \
```

This way the output of make wont be so verbose :)